### ISSUE TITLE
getOrders is not a function

### ISSUE THREAD

**Issue by AdamMYoung** [hendt/ebay-api#96]
Hi, I'm currently implementing this library within a React app. I can authenticate and get a user token just fine, but when it comes to calling getOrders the app crashes, and the console specifies it's not a function.

Here's how I'm calling the API:
```
	useEffect(() => {
		if (isAuthenticated) {
			eBay.sell.fulfillment.getOrders().then((res) => {
				console.log(res);
			});
		}
	}, [isAuthenticated]);
```

And the error:
![image](https://user-images.githubusercontent.com/23509173/141305454-08afeaa9-9afb-4d3b-af8d-5400f8256ed7.png)

**Comment by dantio** [comment]
Weird. Call looks good for me. Is eBay an instance of eBayApi? Are you using the v5.0.0 version?

**Comment by AdamMYoung** [comment]
Yep, on v5.0.0 and is an instance of eBayApi. The code is [here](https://github.com/AdamMYoung/ebay-heuristics-v2), ebayAPi is instantiated in AuthProvider.tsx and exported, and is consumed in Home.tsx. Appreciate your help :)

**Comment by dantio** [comment]
Thank you for the repo. It does work for me (i use npm).
The orders are returned. However, I see `Refused to set unsafe header "Accept-Encoding"` error. I'll try with yarn now.
What happens if you add `console.log(eBay.sell.fulfillment)` ?

**Comment by dantio** [comment]
Also works with yarn. What Node version are you using?

**Comment by AdamMYoung** [comment]
Currently on node 14.17.1, tried both yarn and npm and both failing for me. I've got a deployed version on netlify which is also failing [here](https://ebay-heuristics-v2.netlify.app/), tried on both chrome and safari. Any ideas?

**Comment by dantio** [comment]
I see. This IS strange. the "fulfillment" is actually 'account'. Something broken here: https://github.com/hendt/ebay-api/blob/master/src/api/apiFactory.ts#L117

**Comment by AdamMYoung** [comment]
Ah, could it potentially be due to minification on my end causing the variable names not to match the minified class names, and it resolving to "account" since it's first alphabetically?

**Comment by dantio** [comment]
It's minification, you are right. The RestfulApiClass.name is always 'n'. I think it's better to hardcode the name as property to every Api Class...

**Comment by dantio** [comment]
I have released `v5.0.2`. Please try with this version.

**Comment by AdamMYoung** [comment]
Amazing, thanks for fixing so quickly. All working as expected now
